### PR TITLE
update hardcoded paths to 2016

### PR DESCRIPTION
**Pull request by OlivierBondu**
should be painless

yet to do:
- [x] assess efficiency for the various datasets on runs < 274094

### CODE REVIEW

**Comment by OlivierBondu** [comment]
Efficiency for Double Mu on Run2016B is 99.6602%, looking good

File: plugins/Categories.cc [hunk 1]
```diff
@@ -43,18 +43,14 @@ bool MuMuCategory::event_in_category_post_analyzers(const ProducersManager& prod
...
 };
 
 void MuMuCategory::evaluate_cuts_post_analyzers(CutManager& manager, const ProducersManager& producers, const AnalyzersManager& analyzers) const {
     const HLTProducer& hlt = producers.get<HLTProducer>("hlt");
     for (const std::string& path: hlt.paths) 
     {
-        if (path.find("HLT_Mu17_TrkIsoVVL_Mu8_TrkIsoVVL_DZ_v") != std::string::npos) manager.pass_cut("fire_trigger_Mu17_Mu8");
-        if (path.find("HLT_Mu17_TrkIsoVVL_TkMu8_TrkIsoVVL_DZ_v") != std::string::npos) manager.pass_cut("fire_trigger_Mu17_TkMu8");
-        if (path.find("HLT_IsoMu27_v") != std::string::npos) manager.pass_cut("fire_trigger_IsoMu27");
+        if (path.find("HLT_Mu") != std::string::npos) manager.pass_cut("fire_trigger_MuMu");
```
Comment (blinkseb) [inline plugins/Categories.cc]: This condition is also true for MuEl paths, which mean an event will be in two categories at the same time (not sure it's possible?). It'd probably be easier to use a regex here, something like `HLT_Mu.*_Mu`

**Review by blinkseb — CHANGES_REQUESTED** [review]

**Review by blinkseb — APPROVED** [review]

**Comment by blinkseb** [comment]
I updated the PR using regex to match trigger paths.

**Comment by blinkseb** [comment]
PR updated. I added support for multiple efficiency for a given HLT path / leg (like Double Mu ones). Luminosity values in the python config files are not the correct ones and must be updated.

For a given path, you can now specify more than one JSON file. Each file is assigned a weight. This weight will be used to create a discrete probability function, where each path is assigned the probability `weight / ∑(weights)`. When requesting an efficiency, one is chose randomly based on this distribution.

**Comment by BrieucF** [comment]
:+1:  with moving the modifications to `triggers.xml` in this PR.

**Comment by blinkseb** [comment]
`triggers.xml` added. Merging!

**Comment by OlivierBondu** [comment]
thanks !

**Pull request merged**